build.gradle

```diff
@@ -0,0 +1,158 @@
+import org.apache.tools.ant.filters.ReplaceTokens
+
+
+plugins {
+    id 'war'
+    id 'java'
+    // IntelliJ IDEA plugin here to allow integration tests to appear properly in IDEs.
+    id 'idea'
+}
+
+repositories {
+    mavenLocal()
+    jcenter()
+
+    maven {
+        url = 'http://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/m2repo'
+    }
+}
+
+dependencies {
+
+    runtime 'jstl:jstl:1.+'
+
+    implementation 'org.apache.logging.log4j:log4j-core:[2.13.0,3.0)'
+    implementation 'org.apache.logging.log4j:log4j-slf4j-impl:[2.13.0,3.0)'
+    implementation 'com.opencsv:opencsv:[5.1,6.0)'
+    implementation 'org.opencadc:cadc-web-util:[1.2.7,1.3.0)'
+    implementation 'commons-net:commons-net:3.6'
+
+    runtime 'javax.servlet.jsp.jstl:jstl-api:1.2'
+    runtime 'javax.servlet:jstl:1.2'
+    runtime 'org.apache.taglibs:taglibs-standard-impl:1.2.5'
+
+    testCompile 'ca.nrc.cadc:web-test:[2.0, )'
```

Comment:
The `xxx[cC]ompile` tasks have been deprecated, so switch those to `implementation` tasks.  See https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation.

Reply:
done